projects/storefrontstyles/scss/themes/calydon/_cart-item.scss

```diff
@@ -0,0 +1,7 @@
+%cx-cart-item-calydon {
```

Comment:
I reckon that inside each time you might not only have components but maybe also `cx-base` stylesheets so you probalbly should add a structure inside calydon that will look something like:
`themes/calydon/components/`
`themes/calydon/layout/`
`themes/calydon/cxbase/` and so one. You can create only component for now I guess.